### PR TITLE
feat: (moderate-reaction-sounds) enable moderator to mute reaction sounds

### PR DESCRIPTION
**Pull request by andioltean**
add "Mute reaction sounds for everyone" option for moderator in settings
- used commands to send info to back-end and listen for incoming commands
pull out moderator settings from "More" settings tab and added "Moderator" tab in settings
add option in the footer menu for Participants pane to go to moderator settings
add option for moderator to disable all reaction sounds the reaction sounds notification
increase the time for first sound notification by 2 seconds so a user has more time to disable sounds
refactor "customActionNameKey" and "customActionHandler" to be an array instead of a single value

added unit tests fixes here https://github.com/jitsi/jitsi-meet-torture/pull/459

moderator_settings
more_moderation_control_link

### CODE REVIEW

File: react/features/reactions-muted/constants.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
```
Comment (quitrk) [inline react/features/reactions-muted/constants.js:1]: i don't think there's need for a separate `reactions-muted` structure, since is just a part of the reactions features

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by andioltean** [comment]
Jenkins please test this please.

**Comment by vp8x8** [comment]
Jenkins test this please.

**Comment by saghul** [comment]
@andioltean Hey andi, can you rebase this please?

**Comment by andioltean** [comment]
Please jenkins test this please

**Comment by andioltean** [comment]
Please jenkins test this please

File: react/features/base/devices/middleware.js [hunk 2]
```diff
@@ -132,7 +134,7 @@ MiddlewareRegistry.register(store => next => action => {
             description: additionalCameraErrorMsg,
             descriptionKey: cameraErrorMsg,
             titleKey
-        }, NOTIFICATION_TIMEOUT_TYPE.MEDIUM));
+        }, WARNING_DISPLAY_TIMER));
```
Comment (robertpin) [inline react/features/base/devices/middleware.js:137]: Why was this changed?

Comment (saghul) [reply react/features/base/devices/middleware.js:137]: Good point, it shouldn't.

File: react/features/reactions/constants.js [hunk 3]
```diff
@@ -14,6 +14,13 @@ import {
  */
 export const ENDPOINT_REACTION_NAME = 'endpoint-reaction';
 
+/**
+ * The (name of the) command which transports the state (represented by
+ * {State} for the local state at the time of this writing) of a {FollowMe}
```
Comment (robertpin) [inline react/features/reactions/constants.js:19]: Update comment please

File: react/features/settings/components/web/ModeratorTab.js [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const content = this._renderModeratorSettings();
```
Comment (robertpin) [inline react/features/settings/components/web/ModeratorTab.js:78]: I don't think this variable is needed

File: react/features/settings/functions.js [hunk 5]
```diff
@@ -79,14 +79,30 @@ export function normalizeUserInputURL(url: string) {
...
-export function shouldShowOnlyDeviceSelection() {
-    return interfaceConfig.SETTINGS_SECTIONS.length === 1
-        && isSettingEnabled('devices');
+export function getMoreTabProps(stateful: Object | Function) {
+    const state = toState(stateful);
+    const framerate = state['features/screen-share'].captureFrameRate ?? SS_DEFAULT_FRAME_RATE;
+    const language = i18next.language || DEFAULT_LANGUAGE;
+    const configuredTabs = interfaceConfig.SETTINGS_SECTIONS || [];
+
+    // The settings sections to display.
```
Comment (robertpin) [inline react/features/settings/functions.js:95]: Remove leftover comment

**Review by Calinteodor — DISMISSED** [review]

**Comment by saghul** [comment]
@Calinteodor Please take another look tomorrow, we must not remove the notification timeout types.

File: react/features/av-moderation/middleware.js [hunk 6]
```diff
@@ -221,9 +220,9 @@ StateListenerRegistry.register(
                     dispatch(showNotification({
                         titleKey: 'notify.hostAskedUnmute',
                         sticky: true,
-                        customActionNameKey: 'notify.unmute',
-                        customActionHandler: () => dispatch(muteLocal(false, MEDIA_TYPE.AUDIO))
-                    }, NOTIFICATION_TIMEOUT_TYPE.STICKY));
```
Comment (saghul) [inline react/features/av-moderation/middleware.js:226]: (here and elsewhere) Don't remove the timeout.

**Review by saghul — DISMISSED** [review]

**Comment by andioltean** [comment]
Jenkins please test this please.

**Comment by Calinteodor** [comment]
> @Calinteodor Please take another look tomorrow, we must not remove the notification timeout types.

@saghul All the requests have been resolved, tested the PR a bunch of times with different edge cases, this is good to go.

**Review by Calinteodor — APPROVED** [review]
